src/sections/Planning/PlanningCard.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./PlanningCard.scss";
+import PlanningCardHover from './PlanningCardHover';
+
+const PlanningCard = ({image,title,description, children}) => {
+  const [style, setStyle] = React.useState({display: 'block'});
```

Comment:
redundant states. u can have one state that toggles between hover and block state. will also reduce the handlers from two to just one and will make it a bit cleaner

Reply:
Done